libopenage/cvar/cvar.h

```diff
@@ -0,0 +1,43 @@
+// Copyright 2016-2016 the openage authors. See copying.md for legal info.
+
+#pragma once
+
+#include <functional>
+// pxd: from libcpp.string cimport string
+#include <string>
+#include <unordered_map>
+#include <utility>
+
+// pxd: from libopenage.pyinterface.functional cimport PyIfFunc0
+#include "../pyinterface/functional.h"
+
+namespace openage {
+namespace cvar {
+
+using get_func = std::function<std::string()>;
+using set_func = std::function<void(std::string)>;
+
+/**
+ * pxd:
+ *
+ * cppclass CVarManager:
+ *
+ *     string get(string name) except +
+ *     void set(string name, string value) except +
+ */
+class CVarManager {
+	public :
+		bool create(const std::string &name, const std::pair<get_func, set_func> accessors);
+		std::string get(const std::string &name) const;
+		void set(const std::string &name, const std::string &value) const;
+	private :
+		std::unordered_map<std::string, std::pair<get_func, set_func>> store;
```

Comment:
would be great if you could adhere to our current styleguide, the access labels at the same indent level of the class and the members at one tab indent.